src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionPlanJobEndpointUnitTest.java

```diff
@@ -290,76 +283,4 @@ public void findActionPlanJob() throws Exception {
             jsonPath(
                 "$.updatedDateTime", is(new DateMatcher(UPDATED_DATE_TIME_ACTION_PLAN_JOB_ID_1))));
   }
-
-  /**
-   * A Test to execute an action plan providing incorrect json
-   *
-   * @throws Exception exception thrown when postJson does
-   */
-  /*  @Test
-  public void executeActionPlanIncorrectJsonProvided() throws Exception {
-    ResultActions actions = mockMvc.perform(postJson(String.format("/actionplans/%s/jobs", ACTIONPLANID),
-            ACTION_PLAN_JOB_INCORRECT_JSON));
-
-    actions.andExpect(status().isBadRequest())
-            .andExpect(handler().handlerType(ActionPlanJobEndpoint.class))
-            .andExpect(handler().methodName("executeActionPlan"))
-            .andExpect(jsonPath("$.error.code", is(CTPException.Fault.VALIDATION_FAILED.name())))
-            .andExpect(jsonPath("$.error.message", is(PROVIDED_JSON_INCORRECT)))
-            .andExpect(jsonPath("$.error.timestamp", isA(String.class)));
-  }*/
-
-  /**
-   * A Test to execute an action plan providing json failing validation
-   *
-   * @throws Exception exception thrown when postJson does
-   */
-  @Test
-  public void executeActionPlanJsonFailingValidation() throws Exception {
-    final ResultActions actions =
-        mockMvc.perform(
-            postJson(
-                String.format("/actionplans/%s/jobs", ACTIONPLANID),
-                ACTION_PLAN_JOB_JSON_FAILING_VALIDATION));
-
-    actions
-        .andExpect(status().isBadRequest())
-        .andExpect(handler().handlerType(ActionPlanJobEndpoint.class))
-        .andExpect(handler().methodName("executeActionPlan"))
-        .andExpect(jsonPath("$.error.code", is(CTPException.Fault.VALIDATION_FAILED.name())))
-        .andExpect(jsonPath("$.error.message", is(INVALID_JSON)))
-        .andExpect(jsonPath("$.error.timestamp", isA(String.class)));
-  }
```

Comment:
Isn't this a potentially valid test?

Reply:
This endpoint has been updated to not accept JSON input anymore. 
It wasn't being used for anything I could see